proto/osmosis/gamm/v1beta1/tx.proto

```diff
@@ -3,16 +3,17 @@ package osmosis.gamm.v1beta1;
 
 import "gogoproto/gogo.proto";
 import "cosmos/base/v1beta1/coin.proto";
+import "osmosis/swaprouter/v1beta1/swap_route.proto";
 
 option go_package = "github.com/osmosis-labs/osmosis/v13/x/gamm/types";
 
 service Msg {
   rpc JoinPool(MsgJoinPool) returns (MsgJoinPoolResponse);
   rpc ExitPool(MsgExitPool) returns (MsgExitPoolResponse);
   rpc SwapExactAmountIn(MsgSwapExactAmountIn)
-      returns (MsgSwapExactAmountInResponse);
+      returns (MsgSwapExactAmountInResponse) {}
   rpc SwapExactAmountOut(MsgSwapExactAmountOut)
-      returns (MsgSwapExactAmountOutResponse);
+      returns (MsgSwapExactAmountOutResponse) {}
```

Comment:
Why this change?

Reply:
was trying to deprecate those and failed to revert properly. Going to fix